runtime/oti/j9hookable.h

```diff
@@ -25,4 +25,6 @@
 
 #include "omrhookable.h"
 
+#define J9_EVENT_CAN_BE_HOOKED(javaVM, J9_EVENT_MASK) (J9_EVENT_IS_HOOKED(javaVM->hookInterface, J9_EVENT_MASK) || J9_EVENT_IS_RESERVED(javaVM->hookInterface, J9_EVENT_MASK))
```

Comment:
Why is this added to OpenJ9 rather than OMR?

Reply:
Partly to minimize the number of PRs needed, but also because usually if the name has `J9` in it I try as much as possible to keep in the openj9 codebase. However, if you think I should move it to OMR I can do that and open that PR.

Reply:
I think it should be in OMR. The macros J9_EVENT_IS_HOOKED and J9_EVENT_IS_RESERVED are defined in omrhookable.h

Reply:
OK, will do (pending GAC's blessing wrt the rest of the changes in this PR).